### ISSUE TITLE
Click event on underlying hidden field should focus editor

### ISSUE THREAD

**Issue by urbanautomaton** [futurelearn/markdown-editor#1]
In [`GroupWorkspaceForm` we have a visually-hidden `label`](https://github.com/futurelearn/futurelearn/blob/1fc61dff93f495facda2de41f78f64745f8507fc/app/javascript/components/Partials/GroupWorkspaceContributions/Form.jsx#L85-L93) whose purpose appears to be screenreader accessibility; when clicked, it focuses the editor (using functionality provided by this package) and sets the cursor position (using custom prosemirror trickery).

However, the jsx-a11y eslint rules complain if you add click handlers to non-interactive elements, including `label`. This seems fair enough.

Ideally we'd apply this handler to the label's related input element, which also receives a click event if its associated label is clicked, and is an interactive element.  In this case the related input element is a [hidden input rendered by the markdown-editor tool](https://github.com/futurelearn/markdown-editor/blob/6e1150717b377faa161a9a4721b0d6966f0873a7/src/index.tsx#L115-L120).

It seems reasonable to build this functionality into markdown-editor; if the hidden field receives a click event, that means someone's clicked on an associated `label`, so we should focus the editor. This means clients can use the editor more like a regular `input` element and still have expectations met.

The editor [does expose a focus() method](https://github.com/futurelearn/markdown-editor/blob/6e1150717b377faa161a9a4721b0d6966f0873a7/src/index.tsx#L69) (I believe this is provided by prosemirror?), however `GroupWorkspaceForm` doesn't purely use this; it also [does some extra prosemirror magic to set the cursor to the start of the text box](https://github.com/futurelearn/futurelearn/commit/a09e71a6cc2096aba2118ce3c6a9703b13cb8fa2). Presumably that's perfectly doable here, too.

In summary: let's do that thing that I said.
